beetmoverscript/script.py

```diff
@@ -30,16 +31,30 @@
     context.task = get_task(context.config)  # e.g. $cfg['work_dir']/task.json
     # 2. validate the task
     validate_task_schema(context)
-    # 3. generate manifest
+    # 3 grab manifest props with all the useful data
+    context.properties = await get_props(context)
```

Comment:
does this need to be async? I think since this only downloads one file and everything up till now is sequential, I'm not sure if we will see a benefit from concurrency here.

On the other hand, I suppose scriptworker could be doing background things and would appreciate the non-blocking io download. Plus if you removed the async/await, you would need to use something other than retry_request. So I don't think it's worth addressing.